README.md

```diff
@@ -6,6 +6,8 @@ The PatternFly Org is the source for the official documentation for PatternFly 3
 
 Development setup requires yarn. If you do not already have yarn installed on your system, see https://yarnpkg.com/en/.
 
+Node v14.17.6 is also required.
```

Comment:
It looks like the build works with Node `13.7 - 14.19.2` so there's not specific requirement for `14.17.6` as long as a version within this range is used, can you update?